### PR TITLE
Generalized BTCETH deployment. Moved deploy token params to contractN…

### PR DESCRIPTION
**Pull request by bweick**
…ames file. Define allocation bounds as difference from ideal allocation, and create helper function to calculate bounds submitted to contract.

### CODE REVIEW

File: deployments/contractNames.ts [hunk 1]
```diff
@@ -1,11 +1,183 @@
-export const DEPLOYED_TOKEN = {
-  BitEthInitialCollateralSet: 'BitEthInitialCollateralSet',
-  BitEthRebalancingSetToken: 'BitEthRebalancingSetToken',
-  ETHDaiInitialCollateralSet: 'ETHDaiInitialCollateralSet',
-  ETHDaiRebalancingSetToken: 'ETHDaiRebalancingSetToken',
-  BTCDaiInitialCollateralSet: 'BTCDaiInitialCollateralSet',
-  BTCDaiRebalancingSetToken: 'BTCDaiRebalancingSetToken',
-};
+import BigNumber from 'bignumber.js';
```
Comment (felix2feng) [inline deployments/contractNames.ts]: Is there a name for this file that reflects what it is better what it is?

File: types/deployment_stage_interface.ts [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+}
+
+interface NetworkConstants {
+	production: any;
+	staging: any;
+	development: any;
+}
+
+export interface DeployedSetInfo {
+	PRICE_PRECISION: BigNumber;
```
Comment (felix2feng) [inline types/deployment_stage_interface.ts]: Extra Tab

File: types/deployment_stage_interface.ts [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
   deploy(web3: any): Promise<any>;
+}
+
+interface NetworkConstants {
+	production: any;
+	staging: any;
+	development: any;
+}
+
+export interface DeployedSetInfo {
```
Comment (felix2feng) [inline types/deployment_stage_interface.ts]: 💯 Good stuff adding the typing

File: types/deployment_stage_interface.ts [hunk 4]
```diff
@@ -1,3 +1,26 @@
+import BigNumber from 'bignumber.js';
+
 export interface DeploymentStageInterface {
   deploy(web3: any): Promise<any>;
+}
+
+interface NetworkConstants {
+	production: any;
```
Comment (felix2feng) [inline types/deployment_stage_interface.ts]: Think these should be numbers vs any type

Comment (bweick) [reply types/deployment_stage_interface.ts]: Disagree cause there might be other kinds of constants we want to tweak in the future that aren't limited to numbers. Could be addresses and what not.

**Pull request merged**